package.json

```diff
@@ -51,7 +51,7 @@
     "@mollie/eslint-config-typescript": "^1.6.5",
     "@types/jest": "^29.5.11",
     "@types/node": "^18.14.6",
-    "axios-mock-adapter": "1.19.0",
+    "@types/node-fetch": "^2.6.11",
```

Comment:
The `@types` and the regular dependency are out of sync.

Reply:
It's the same major and at the time of writing there is no newer types package for node-fetch.
This should be fine.